podcasts/End of Year/Stories/ListeningTimeStory.swift

```diff
@@ -13,37 +13,49 @@ struct ListeningTimeStory: ShareableStory {
-                    if NSLocale.isCurrentLanguageEnglish {
-                        StoryLabel(L10n.eoyStoryListenedToUpdated("\n\(time)\n"), highlighting: [time], for: .title)
-                    } else {
-                        StoryLabel(L10n.eoyStoryListenedTo("\n\(time)\n"), highlighting: [time], for: .title)
-                    }
-                    StoryLabel(FunMessage.timeSecsToFunnyText(listeningTime), for: .subtitle)
-                        .opacity(0.8)
+            PodcastCoverContainer(geometry: geometry) {
+                StoryLabelContainer(geometry: geometry) {
+                    StoryLabel(L10n.eoyStoryListenedToTitle, for: .title, geometry: geometry)
+                    StoryLabel(L10n.eoyStoryListenedToSubtitle, for: .subtitle, color: Color(hex: "8F97A4"), geometry: geometry)
                 }
 
-                // Podcast images angled to fill the width of the view
-                let size = 0.30 * geometry.size.height
-
-                HStack(spacing: 20) {
-                    ForEach(Constants.displayedPodcasts, id: \.self) {
-                        podcastCover($0, size: size)
+                // We treat numbers below 21 differently, as there's much more space available
+                let maxHeight = Int(listeningTime.firstNumber) ?? 0 <= 21 ? 0.5 : 0.35
+                let subtitleTopPadding = (Int(listeningTime.firstNumber) ?? 0 <= 21 ? 0.35 : 0.25)
+                Text(listeningTime.firstNumber)
+                    .font(.custom("DM Sans", size: geometry.size.height * 0.4))
+                    .fontWeight(.regular)
+                    .frame(width: geometry.size.width - 70)
+                    .frame(maxHeight: geometry.size.height * maxHeight)
+                    .scaledToFill()
+                    .minimumScaleFactor(0.5)
+                    .lineLimit(1)
+                    .foregroundColor(.white)
+                    .padding([.leading, .trailing], 35)
+                    .overlay {
+                        StoryLabel(listeningTime.subtitle, for: .subtitle, color: Color(hex: "8F97A4"), geometry: geometry)
+                            .padding(.top, geometry.size.height * subtitleTopPadding)
                     }
-                }
-                .applyPodcastCoverPerspective()
-                .padding(.top)
-            }.frame(width: geometry.size.width)
-        }.background(DynamicBackgroundView(podcast: podcasts[0]))
-    }
 
-    @ViewBuilder
-    func podcastCover(_ index: Int, size: Double) -> some View {
-        let podcast = podcasts[safe: index] ?? podcasts[0]
-        PodcastCover(podcastUuid: podcast.uuid)
-            .frame(width: size, height: size)
+                Spacer()
+
+                CircleDays(proxy: geometry, listeningTime: listeningTime)
+
+                Spacer()
+
+                Rectangle()
+                    .frame(height: geometry.size.height * 0.1)
+                    .opacity(0)
```

Comment:
hmm. What's this doing?

Reply:
This is the Rectangle that fills the "non-listened" days. So, if a user listened to 2 days, this will fill the remaining space for the 5 circles (given there are 7). Added a comment about it in 04f512085b054ac9647354dea1c509b29352c10a